prov/verbs/src/verbs_eq.c

```diff
@@ -110,11 +110,20 @@ struct fi_ibv_xrc_ep *fi_ibv_eq_xrc_conn_tag2ep(struct fi_ibv_eq *eq,
 
 	index = ofi_key2idx(&eq->xrc.conn_key_idx, (uint64_t)conn_tag);
 	ep = ofi_idx_lookup(eq->xrc.conn_key_map, index);
-	if (!ep || !ep->conn_setup || (ep->conn_setup->conn_tag != conn_tag)) {
+	if (!ep || ep->magic != VERBS_XRC_EP_MAGIC) {
```

Comment:
Does an EP get the magic field cleared after close? Just curious for the case where an EP could be reused.

Reply:
It does.